code/style.css

```diff
@@ -1,5 +1,248 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+body { 
+  background-color:gainsboro;
+  color:black;
+  font-family: Helvetica, sans-serif;
+  margin: 0;
+}
+
+h1 {
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  font-size: 40px;
+  line-height: 30px;
+  color:black;
+}
+
+h2 {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif
+}
+
+h3 {
+  font-size: 26px;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
+h4 {
+font-size: 32px;
+font-weight: 100;
+margin-top: -16px;
+font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+margin-bottom: 16px;
+}
 
-body {
-  background-color: peachpuff;
+.sub {
+  font-family: impact, 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 24px;
+  font-weight: 500;
 }
+
+p {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-decoration: none;
+  color: black;
+}
+
+.container {
+  width: 100%;
+  flex-direction: column; 
+  max-width: 2000px;
+  text-align: center;
+}
+
+footer {
+  text-align: center;
+  text-decoration: none;
+  background-color: gainsboro;
+  color: black;
+}
+
+/*** HEADER ***/
+
+header {
+  background: white;
+  height: 80px;
+  display: flex; 
+  align-items: center;
+  padding: 0 16px;}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+}
+
+.hamburger div {
+width: 30px;
+height: 5px;
+background: black;
+border-radius: 5px;
+}
+
+header img {
+  width: 100px;
+  height: auto;
+  margin: 5px;
+}
+nav { 
+display: none; 
+}
+
+nav a {
+  color: black;
+  text-decoration: none;
+  margin-left: 20px;
+  font-family: monospace;
+  font-size: 15px;
+  
+}
+
+/*** FIRST CONTAINER ***/
+
+.first-container .container {
+  padding: 20px;
```

Comment:
This padding causes a horziontal scroll on your page which we want to avoid when we work with responsive web design. Change this row to `padding: 20px 0;` to keep the padding to the top/bottom of the element, but have 0 padding on the sides